# PointCloudWithGrasps.msg
Header header
sensor_msgs/PointCloud2 point_cloud
Grasp[] grasps

